src/functionalTest/java/uk/gov/hmcts/reform/professionalapi/FindUsersStatusByEmailTest.java

```diff
@@ -104,16 +113,21 @@ public void rdcc_719_ac1_find_user_status_by_email_with_caseworker_publiclaw_cou
         userRoles.add("pui-organisation-manager");
         NewUserCreationRequest userCreationRequest = createUserRequest(userRoles);
 
-        // creating user in idam with the same email used in the invite user so that status automatically will update in the up
-        professionalApiClient.getMultipleAuthHeadersExternal(puiOrgManager, userCreationRequest.getFirstName(), userCreationRequest.getLastName(), userCreationRequest.getEmail());
+        // creating user in idam with the same email used in the invite user so that status automatically will update
+        // in the up

```

Comment:
This comment would read better if it was split earlier, suggestion:

```
        // creating user in idam with the same email used in the invite user
        // so that status automatically will update in the up
```